src/IssueViz.Security.UnitTests/OpenInIDE/Api/OpenInIDERequestHandlerTests.cs

```diff
@@ -17,333 +17,332 @@
-//                 mock.VerifyAll();
-//                 mock.Invocations.Count.Should().Be(1);
-//             }
-//         }
-//
-//         private static void CheckNotCalled(params Mock[] mocks)
-//         {
-//             foreach(var mock in mocks)
-//             {
-//                 Console.WriteLine($"Checking mock wasn't called: {mock.Object.GetType()}");
-//                 mock.Invocations.Should().BeEmpty();
-//             }
-//         }
-//
-//         private void CheckInfoBarCleared() =>
-//             failureInfoBarMock.Verify(x => x.ClearAsync(), Times.Once);
-//
-//         private void CheckInfoBarNotShown() =>
-//             failureInfoBarMock.Verify(x => x.ShowAsync(It.IsAny<Guid>()), Times.Never);
-//
-//         private void CheckInfoBarShown() =>
-//             failureInfoBarMock.Verify(x => x.ShowAsync(HotspotsToolWindow.ToolWindowId), Times.Once);
-//
-//         private class TestShowHotspotRequest : IShowHotspotRequest
-//         {
-//             public Uri ServerUrl { get; set; }
-//             public string OrganizationKey { get; set; }
-//             public string ProjectKey { get; set; }
-//             public string HotspotKey { get; set; }
-//         }
-//     }
-// }
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Core.Telemetry;
+using SonarLint.VisualStudio.Integration;
+using SonarLint.VisualStudio.IssueVisualization.Editor;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Api;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Contract;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE_Hotspots;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE_Hotspots.HotspotsList;
+using SonarLint.VisualStudio.IssueVisualization.Selection;
+using SonarLint.VisualStudio.TestInfrastructure;
+using SonarQube.Client;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.OpenInIDE.Api
+{
+    [TestClass]
+    public class OpenInIDERequestHandlerTests_MEF
+    {
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
+        {
+            MefTestHelpers.CheckTypeCanBeImported<OpenInIDERequestHandler, IOpenInIDERequestHandler>(
+                MefTestHelpers.CreateExport<IIDEWindowService>(),
+                MefTestHelpers.CreateExport<IToolWindowService>(),
+                MefTestHelpers.CreateExport<IOpenInIDEStateValidator>(),
+                MefTestHelpers.CreateExport<ISonarQubeService>(),
+                MefTestHelpers.CreateExport<IHotspotToIssueVisualizationConverter>(),
+                MefTestHelpers.CreateExport<ILocationNavigator>(),
+                MefTestHelpers.CreateExport<IOpenInIDEHotspotsStore>(),
```

Comment:
Changed the type names used e.g. `IHotspotsStore` -> `IOpenInIDEHotspotsStore`

No other changes in this file.